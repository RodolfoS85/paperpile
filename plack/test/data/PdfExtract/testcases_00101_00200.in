---
file: ../../../PDFREPOSITORY/000101.pdf
---
file: ../../../PDFREPOSITORY/000102.pdf
---
file: ../../../PDFREPOSITORY/000103.pdf
---
file: ../../../PDFREPOSITORY/000104.pdf
---
file: ../../../PDFREPOSITORY/000105.pdf
---
file: ../../../PDFREPOSITORY/000106.pdf
---
file: ../../../PDFREPOSITORY/000107.pdf
---
file: ../../../PDFREPOSITORY/000108.pdf
---
file: ../../../PDFREPOSITORY/000109.pdf
---
file: ../../../PDFREPOSITORY/000110.pdf
---
file: ../../../PDFREPOSITORY/000111.pdf
---
file: ../../../PDFREPOSITORY/000112.pdf
---
file: ../../../PDFREPOSITORY/000113.pdf
---
file: ../../../PDFREPOSITORY/000114.pdf
---
file: ../../../PDFREPOSITORY/000115.pdf
---
file: ../../../PDFREPOSITORY/000116.pdf
---
file: ../../../PDFREPOSITORY/000117.pdf
---
file: ../../../PDFREPOSITORY/000118.pdf
---
file: ../../../PDFREPOSITORY/000119.pdf
---
file: ../../../PDFREPOSITORY/000120.pdf
---
file: ../../../PDFREPOSITORY/000121.pdf
---
file: ../../../PDFREPOSITORY/000122.pdf
---
file: ../../../PDFREPOSITORY/000123.pdf
---
file: ../../../PDFREPOSITORY/000124.pdf
---
file: ../../../PDFREPOSITORY/000125.pdf
---
file: ../../../PDFREPOSITORY/000126.pdf
---
file: ../../../PDFREPOSITORY/000127.pdf
---
file: ../../../PDFREPOSITORY/000128.pdf
---
file: ../../../PDFREPOSITORY/000129.pdf
---
file: ../../../PDFREPOSITORY/000130.pdf
---
file: ../../../PDFREPOSITORY/000131.pdf
---
file: ../../../PDFREPOSITORY/000132.pdf
---
file: ../../../PDFREPOSITORY/000133.pdf
---
file: ../../../PDFREPOSITORY/000134.pdf
---
file: ../../../PDFREPOSITORY/000135.pdf
---
file: ../../../PDFREPOSITORY/000136.pdf
---
file: ../../../PDFREPOSITORY/000137.pdf
---
file: ../../../PDFREPOSITORY/000138.pdf
---
file: ../../../PDFREPOSITORY/000139.pdf
---
file: ../../../PDFREPOSITORY/000140.pdf
---
file: ../../../PDFREPOSITORY/000141.pdf
---
file: ../../../PDFREPOSITORY/000142.pdf
---
file: ../../../PDFREPOSITORY/000143.pdf
---
file: ../../../PDFREPOSITORY/000144.pdf
---
file: ../../../PDFREPOSITORY/000145.pdf
---
file: ../../../PDFREPOSITORY/000146.pdf
---
file: ../../../PDFREPOSITORY/000147.pdf
---
file: ../../../PDFREPOSITORY/000148.pdf
---
file: ../../../PDFREPOSITORY/000149.pdf
---
file: ../../../PDFREPOSITORY/000150.pdf
---
file: ../../../PDFREPOSITORY/000151.pdf
---
file: ../../../PDFREPOSITORY/000152.pdf
---
file: ../../../PDFREPOSITORY/000153.pdf
---
file: ../../../PDFREPOSITORY/000154.pdf
---
file: ../../../PDFREPOSITORY/000155.pdf
---
file: ../../../PDFREPOSITORY/000156.pdf
---
file: ../../../PDFREPOSITORY/000157.pdf
---
file: ../../../PDFREPOSITORY/000158.pdf
---
file: ../../../PDFREPOSITORY/000159.pdf
---
file: ../../../PDFREPOSITORY/000160.pdf
---
file: ../../../PDFREPOSITORY/000161.pdf
---
file: ../../../PDFREPOSITORY/000162.pdf
---
file: ../../../PDFREPOSITORY/000163.pdf
---
file: ../../../PDFREPOSITORY/000164.pdf
---
file: ../../../PDFREPOSITORY/000165.pdf
---
file: ../../../PDFREPOSITORY/000166.pdf
---
file: ../../../PDFREPOSITORY/000167.pdf
---
file: ../../../PDFREPOSITORY/000168.pdf
---
file: ../../../PDFREPOSITORY/000169.pdf
---
file: ../../../PDFREPOSITORY/000170.pdf
---
file: ../../../PDFREPOSITORY/000171.pdf
---
file: ../../../PDFREPOSITORY/000172.pdf
---
file: ../../../PDFREPOSITORY/000173.pdf
---
file: ../../../PDFREPOSITORY/000174.pdf
---
file: ../../../PDFREPOSITORY/000175.pdf
---
file: ../../../PDFREPOSITORY/000176.pdf
---
file: ../../../PDFREPOSITORY/000177.pdf
---
file: ../../../PDFREPOSITORY/000178.pdf
---
file: ../../../PDFREPOSITORY/000179.pdf
---
file: ../../../PDFREPOSITORY/000180.pdf
---
file: ../../../PDFREPOSITORY/000181.pdf
---
file: ../../../PDFREPOSITORY/000182.pdf
---
file: ../../../PDFREPOSITORY/000183.pdf
---
file: ../../../PDFREPOSITORY/000184.pdf
---
file: ../../../PDFREPOSITORY/000185.pdf
---
file: ../../../PDFREPOSITORY/000186.pdf
---
file: ../../../PDFREPOSITORY/000187.pdf
---
file: ../../../PDFREPOSITORY/000188.pdf
---
file: ../../../PDFREPOSITORY/000189.pdf
---
file: ../../../PDFREPOSITORY/000190.pdf
---
file: ../../../PDFREPOSITORY/000191.pdf
---
file: ../../../PDFREPOSITORY/000192.pdf
---
file: ../../../PDFREPOSITORY/000193.pdf
---
file: ../../../PDFREPOSITORY/000194.pdf
---
file: ../../../PDFREPOSITORY/000195.pdf
---
file: ../../../PDFREPOSITORY/000196.pdf
---
file: ../../../PDFREPOSITORY/000197.pdf
---
file: ../../../PDFREPOSITORY/000198.pdf
---
file: ../../../PDFREPOSITORY/000199.pdf
---
file: ../../../PDFREPOSITORY/000200.pdf
