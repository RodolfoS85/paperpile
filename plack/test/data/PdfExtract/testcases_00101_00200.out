---
authors: 'Baumstark, Tilman and Schröder, Astrid R. W. and Riesner, Detlev'
title: 'Viroid processing: switch from cleavage to ligation is driven by a change from a tetraloop to a loop E conformation'
---
authors: 'Beato, Miguel and Eisfeld, Karin'
title: Transcription factor access to chromatin
---
authors: 'Becker, Annette and Winter, Kai-Uwe and Meyer, Britta and Saedler, Heinz and Theißen, Gunter'
title: MADS-Box Gene Diversity in Seed Plants 300 Million Years Ago
---
authors: 'Beckstette, Michael and Homann, Robert and Giegerich, Robert and Kurtz, Stefan'
doi: 10.1186/1471-2105-7-389
title: Fast index based algorithms and software for matching position specific scoring matrices
---
authors: 'Beldade, Patricia and Koops, Kees and Brakefield, Paul M.'
doi: 10.1073/pnas.222236199
title: 'Modularity, individuality, and evo-devo in butterfly wings'
---
authors: 'Bell, Charles E and Lewis, Mitchell'
title: 'The Lac repressor: a second generation of structural and functional studies'
---
authors: 'BEN-DOR, Amir and Shamir, Ron and Yakhini, Zohar'
title: Clustering Gene Expression Patterns
---
authors: 'Ben-Gal, I. and Shani, A. and Gohr, A. and Grau, J. and Arviv, S. and Shmilovici, A. and Posch, S. and Grosse, I.'
title: Identification of Transcription Factor Binding Sites with Variable-order Bayesian Networks
---
authors: 'Benson, Gary'
title: 'Tandem repeats finder: a program to analyze DNA sequences'
---
authors: 'Bentley, David'
title: 'The mRNA assembly line: transcription and processing machines in the same factory'
---
authors: 'Berg, Johannes and Lassig, Michael'
doi: 10.1073/pnas.0305199101
title: Local graph alignment and motif search in biological networks
---
authors: 'Berger, B. and King, J. and Schwartz, R. and Shor, P. W.'
title: Local rule mechanism for selecting icosahedral shell geometry
---
authors: 'Berger, Bonnie and Leightont, Tom'
title: Protein Folding in the Hydrophobic-Hydrophilic NP-Complete
---
authors: 'Berkhin, Pavel'
title: Survey of Clustering Data Mining Techniques
---
authors: 'Berman, Piotr and Gupta, Bhaskar Das and Mubayi, Dhruv and Sloan, Robert and Turn, Gyrgy and Zhang, Yi'
title: The Protein Sequence Design Problem in Canonical Model on 2D and 3D Lattices
---
authors: 'Berman, Benjamin P. and Nibu, Yutaka and Pfeiffer, Barret D. and Tomancak, Pavel and Celniker, Susan E. and Levine, Michael and Rubin, Gerald M. and Eisen, Michael B.'
doi: 10.1073/pnas.231608898
title: Exploiting transcription factor binding site clustering to identify cis-regulatory modules involved in pattern formation in the Drosophila genome
---
authors: 'Berman, Benjamin P and Pfeiffer, Barret D and Laverty, Todd R and Salzberg, Steven L and Rubin, Gerald M and Eisen, Michael B and Celniker, Susan E'
title: 'Computational identification of developmental enhancers: conservation and function of transcription factor binding-site clusters in Drosophila melanogaster and Drosophila pseudoobscura'
---
authors: 'Berman, Helen M. and Westbrook, John and Feng, Zukang and Gilliland, Gary and Bhat, T. N. and Weissig, Helge and Shindyalov, Ilya N. and Bourne, Philip E.'
title: The Protein Data Bank
---
authors: 'Bernhart, Stephan H and Hofacker, Ivo L and Will, Sebastian and Gruber, Andreas R and Stadler, Peter F'
doi: 10.1186/1471-2105-9-474
title: 'RNAalifold: improved consensus structure prediction for RNA alignments'
---
authors: 'Bernhart, Stephan H and Tafer, Hakim and Mückstein, Ulrike and Flamm, Christoph and Stadler, Peter F and Hofacker, Ivo L'
doi: 10.1186/1748-7188-1-3
title: Partition function and base pairing probabilities of RNA heterodimers
---
authors: 'Berrera, Marco and Molinari, Henriette and Fogolari, Federico'
title: Amino acid empirical contact energy definitions for fold recognition in the space of contact maps
---
authors: 'Berry, Steven M. and Gieselman, Matt D. and Nilges, Mark J. and van der Donk, Wilfred A. and Lu, Yi'
doi: 10.1021/ja0169163
title: An Engineered Azurin Variant Containing a Selenocysteine Copper Ligand
---
authors: 'Besancon, R. and Prost, A. L. and Konecny, L. and Latour, P. and Petiot, P. and Boutrand, L. and Kopp, N. and Mularoni, A. and Chamba, G. and Vandenberghe, A.'
title: Alternative exon 3 splicing of the human major protein zero gene in white blood cells and peripheral nerve tissue
---
authors: 'Besemer, John and Borodovsky, Mark'
title: Heuristic approach to deriving models for gene finding
---
authors: 'Besemer, John and Lomsadze, Alexandre and Borodovsky, Mark'
title: 'GeneMarkS: a self-training method for prediction of gene starts in microbial genomes. Implications for finding sequence motifs in regulatory regions'
---
authors: 'Bessonov, N. and Volpert, V.'
title: Dynamical Models of Plant Growth
---
authors: 'Bettencourt-Dias, M. and Giet, R. and Sinka, R. and Mazumdar, A. and Lock, W. G. and Balloux, F. and Zafiropoulos, P. J. and Yamaguchi, S. and Winter, S. and Carthew, R. W. and Cooper, M. and Jones, D. and Frenz, L. and Glover, D. M.'
title: Genome-wide survey of protein kinases required for cell cycle progression
---
authors: 'Beyer, Andreas and Workman, Christopher and Hollunder, Jens and Radke, Dorte and Moller, Ulrich and Wilhelm, Thomas and Ideker, Trey'
doi: 10.1371/journal.pcbi.0020070
title: Integrated Assessment and Prediction of Transcription Factor Binding
---
authors: 'Bhattacharyya, Roby P. and Remnyi, Attila and Yeh, Brian J. and Lim, Wendell A.'
doi: 10.1146/annurev.biochem.75.103004.142710
title: 'Domains, Motifs, and Scaffolds: The Role of Modular Interactions in the Evolution and Wiring of Cell Signaling Circuits'
---
authors: 'Bianchi, Marco E. and Beltrame, Monica'
title: "PROTEIN COMPLEXES '98 Flexing DNA: HMG-Box Proteins and Their Partners"
---
authors: 'Biere, Armin and Sinz, Carsten'
title: Decomposing SAT Problems into Connected Components
---
authors: 'Bille, Philip'
title: 'Tree Edit Distance, Alignment Distance and Inclusion'
---
authors: 'Bilu, Yonatan and Barkai, Naama'
doi: 10.1186/gb-2005-6-12-r103
title: The design of transcription-factor binding sites is affected by combinatorial regulation
---
authors: 'Birney, Ewan and Durbin, Richard'
title: Using GeneWise in the Drosophila Annotation Experiment
---
authors: 'Birney, Ewan and Thompson, Julie D. and Gibson, Toby J.'
title: 'PairWise and SearchWise: finding the optimal alignment in a simultaneous comparison of a protein profile against all DNA translation frames'
---
authors: 'Birol, İnanç and Jackman, Shaun D and Nielsen, Cydney and Qian, Jenny Q and Varhol, Richard and Stazyk, Greg and Morin, Ryan D and Zhao, Yongjun and Hirst, Martin and Schein, Jacqueline E and Hors-man, Doug E and Connors, Joseph M and Gascoyne, Randy D and Marra, Marco A and Jones, Steven JM'
title: De novo Transcriptome Assembly with ABySS
---
authors: 'Birringer, Marc and Pilawa, Sandra and Flohé, Leopold'
doi: 10.1039/b205802m
title: Trends in selenium biochemistry
---
authors: 'Bistarelli, Stefano and Bottalico, Marco'
title: Constraint Based Languages for Biological Reactions
---
authors: 'Blackburne, Benjamin P. and Hirst, Jonathan D.'
doi: 10.1063/1.1590310
title: 'Three-dimensional functional model proteins: Structure function and evolution'
---
authors: 'Blackburne, Benjamin P.'
doi: 10.1063/1.2056545
title: Population dynamics simulations of functional model proteins
---
authors: 'Blanchette, Mathieu and Bourque, Guillaume and Sanko, David'
title: Breakpoint Phylogenies
---
authors: 'Blanchette, Mathieu and Sinha, Saurabh'
title: Separating real motifs from their artifacts
---
authors: 'Blanco, Enrique and Messeguer, Xavier and Smith, Temple F. and Guigo, Roderic'
doi: 10.1371/journal.pcbi
title: Transcription Factor Map Alignment of Promoter Regions
---
authors: 'Blin, Guillaume and Fertin, Guillaume and Rusu, Irena and Sinoquet, Christine'
title: Extending the Hardness of RNA Secondary Structure Comparison
---
authors: 'Blin, Guillaume and Fertin, Guillaume and Hermelin, Danny and Vialette, Stphane'
title: Fixed-Parameter Algorithms for Protein Similarity Search Under mRNA Structure Constraints
---
authors: 'Blom, Nikolaj and Gammeltoft, Steen and Brunak, Sùren'
title: Sequence and Structure-based Prediction of Eukaryotic Protein Phosphorylation Sites
---
authors: 'Bloom, Jesse D and Romero, Philip A and Lu, Zhongyi and Arnold, Frances H'
doi: 10.1186/1745-6150-2-17
title: 'Neutral genetic drift can alter promiscuous protein functions, potentially aiding functional evolution'
---
authors: 'Bloom, Jesse D. and Raval, Alpan and Wilke, Claus O.'
doi: 10.1534/genetics.106.061754
title: Thermodynamics of Neutral Protein Evolution
---
authors: 'Bloom, Jesse D. and Silberg, Jonathan J. and Wilke, Claus O. and Drummond, D. Allan and Adami, Christoph and Arnold, Frances H.'
doi: 10.1073/pnas.0406744102
title: Thermodynamic prediction of protein neutrality
---
authors: 'Blouin, Christian and Butt, Davin and Roger, Andrew James'
doi: 10.1110/ps.03299804
title: 'Rapid evolution in conformational space: A study of loop regions in a ubiquitous GTP binding domain'
---
authors: 'Bø, Trond Hellem and Jonassen, Inge'
title: New feature subset selection procedures for classification of expression profiles
---
authors: 'Bock, A. and Forchhammer, K. and Heider, J. and Leinfelder, W. and Sawers, G. and Veprek, B. and Zinoni, F.'
title: 'Selenocysteine: the 21st amino acid'
---
authors: 'Bock, Joel R. and Gough, David A.'
title: Whole-proteome interaction mining
---
authors: 'Boguski, Mark S. and McIntosh, Martin W.'
title: Biomedical informatics for proteomics
---
authors: 'Bommarito, Salvatore and Peyret, Nicolas and Lucia, Jr., John Santa'
title: Thermodynamic parameters for DNA sequences with dangling ends
---
authors: '{THE ATHANASIUS F. BOMPFUNEWERER CONSORTIUM} and Backofen, Rolf and Bernhart, Stephan H. and Flamm, Christoph and Fried, Claudia and Fritzsch, Guido and Hackermuller, Jorg and Hertel, Jana and Hofacker, Ivo L. and Missal, Kristin and Mosig, Axel and Prohaska, Sonja J. and Rose, Dominic and Stadler, Peter F. and Tanzer, Andrea and Washietl, Stefan and Will, Sebastian'
doi: 10.1002/jezb.21130
title: 'RNAs Everywhere: Genome-Wide Annotation of Structured RNAs'
---
authors: 'BORNBERG-BAUER, Erich and Chan, Hue Sun'
title: 'Modeling evolutionary landscapes: Mutational stability, topology, and superfunnels in sequence space'
---
authors: 'Bornes, Stephanie and Boulard, Mathieu and Hieblot, Corinne and Zanibellato, Catherine and Iacovoni, Jason S. and Prats, Herve and Touriol, Christian'
doi: 10.1074/jbc.M308410200
title: Control of the Vascular Endothelial Growth Factor Internal Ribosome Entry Site (IRES) Activity and Translation Initiation by Alternatively Spliced Coding Sequences
---
authors: 'Bortolussi, Luca and Pal, Alessandro Dal and Dovier, Agostino and Fogolari, Federico'
title: Protein Folding Simulation in CCP
---
authors: 'Bortolussi, Luca'
title: Dynamical Compartments in stochastic Concurrent Constraint Programming
---
authors: 'Bossi, Lionello and Figueroa-Bossi, Nara'
doi: 10.1111/j.1365-2958.2007.05829.x
title: A small RNA downregulates LamB maltoporin in Salmonella
---
authors: 'Boucard, David and Toulme, Jean-Jacques and Di Primo, Carmelo'
doi: 10.1021/bi051187f
title: Bimodal Loop-Loop Interactions Increase the Affinity of RNA Aptamers for HIV-1 RNA Structures
---
authors: 'Boue, Stephanie and Letunic, Ivica and Bork, Peer'
doi: 10.1002/bies.10371
title: Alternative splicing and evolution
---
authors: 'Boue, Stephanie and Vingron, Martin and Kriventseva, Evgenia and Koch, Ina'
title: Theoretical analysis of alternative splice forms using computational methods
---
authors: 'Bourdeau, Véronique and Ferbeyre, Gerardo and Pageau, Marie and Paquin, Bruno and Cedergren, Robert'
title: The distribution of RNA motifs in natural sequences
---
authors: 'Bouvet, Philippe and -T. Allain, Frederic H. and Finger, L. David and Dieckmann, Thorsten and Feigon, Juli'
doi: 10.1006/jmbi.2001.4691
title: Recognition of Pre-formed and Flexible Elements of an RNA Stem-Loop by Nucleolin
---
authors: 'Bratt, Eva and Ohman, Marie'
doi: 10.1261/rna.2750803
title: Coordination of editing and splicing of glutamate receptor pre-mRNA
---
authors: 'Bray, Nicolas and Pachter, Lior'
doi: 10.1101/gr.1960404
title: 'MAVID: Constrained Ancestral Alignment of Multiple Sequences'
---
authors: 'Bray, J. E. and Todd, A. E. and Pearl, F. M. G. and Thornton, J. M. and Orengo, C. A.'
title: 'The CATH Dictionary of Homologous Superfamilies (DHS): a consensus approach for identifying distant structural homologues'
---
authors: 'Brazma, Alvis and Jonassen, Inge and Vilo, Jaak and Ukkonen, Esko'
title: Predicting Gene Regulatory Elements in Silico on a Genomic Scale
---
authors: 'Brejov, Broa and Di Marco, Chrysanne and Vina, Tom and Hidalgo, Sandra Romero'
title: Finding Patterns in Biological Sequences
---
authors: 'Brennan, Richard G and Link, Todd M'
doi: 10.1016/j.mib.2007.03.015
title: 'Hfq structure, function and ligand binding'
---
authors: 'Brennecke, Julius and Cohen, Stephen M'
title: Towards a complete description of the microRNA complement of animal genomes
---
authors: 'Brennecke, Julius and Hipfner, David R. and Stark, Alexander and Russell, Robert B. and Cohen, Stephen M.'
title: bantam Encodes a Developmentally Regulated microRNA that Controls Cell Proliferation and Regulates the Proapoptotic Gene hid in Drosophila
---
authors: 'Brett, David and Hanke, Jens and Lehmann, Gerrit and Haase, Sabine and Delbruck, Sebastian and Krueger, Steen and Reich, Jens and Bork, Peer'
title: EST comparison indicates 38% of human mRNAs contain possible alternative splice forms
---
authors: 'Breunig, Markus M. and Kriegel, Hans-Peter and Ng, Raymond T. and Sander, Jörg'
title: 'LOF: Identifying Density-Based Local Outliers'
---
authors: 'Britten, Roy J.'
title: DNA sequence insertion and evolutionary variation in gene regulation
---
authors: 'Brockmann, Regina and Beyer, Andreas and Heinisch, Jurgen J. and Wilhelm, Thomas'
doi: 10.1371/journal.pcbi.0030057
title: 'Posttranscriptional Expression Regulation: What Determines Translation Rates?'
---
authors: 'Broder, Samuel and Venter, J Craig'
title: 'Whole genomes: the foundation of new biology and medicine'
---
authors: 'Bromberg, Sarina and Dill, Ken A.'
title: Side-chain entropy and packing in proteins
---
authors: 'Bruce, Rebecca'
title: A Bayesian Approach to Semi-Supervised Learning
---
authors: 'Bruce, Shirley R. and CAMERON DINGLE, R. W. and Peterson, Martha L.'
doi: 10.1261/rna.5820103
title: 'B-cell and plasma-cell splicing differences: A potential role in regulated immunoglobulin RNA processing'
---
authors: 'Brudno, Michael and Do, Chuong B. and Cooper, Gregory M. and Kim, Michael F. and Davydov, Eugene and Program, Nisc Comparative Sequencing and Green, Eric D. and Sidow, Arend and Batzoglou, Serafim'
doi: 10.1101/gr.926603
title: 'LAGAN and Multi-LAGAN: Efficient Tools for Large-Scale Multiple Alignment of Genomic DNA'
---
authors: 'Brudno, Michael and Gelfand, Mikhail S. and Spengler, Sylvia and Zorn, Manfred and Dubchak, Inna and Conboy, John G.'
title: Computational analysis of candidate intron regulatory elements for tissue-specific alternative pre-mRNA splicing
---
authors: 'Brun, Christine and Chevenet, François and Martin, David and Wojcik, Jérôme and Guénoche, Alain and Jacq, Bernard'
title: Functional classification of proteins for the prediction of cellular function from a protein-protein interaction network
---
authors: 'Brutlag, Douglas and Apaydin, Serkan and Guestrin, Carlos and Hsu, David and Varma, Chris and Singh, Amit and Latombe, Jean-Claude'
title: Using robotics to fold proteins and dock ligands
---
authors: 'Bryngelson, Joseph D. and Onuchic, Jose Nelson and Socci, Nicholas D. and Wolynes, Peter'
title: 'Funnels, Pathways, and the Energy Landscape of Protein Folding: A Synthesis'
---
authors: 'Brynildsen, Mark P. and Tran, Linh M. and Liao, James C.'
title: A Gibbs Sampler for the Identification of Gene Expression and Network Connectivity Consistency
---
authors: 'Buckley, T. R. and Simon, C. and Flook, P. K. and Misof, B.'
title: Secondary structure and conserved motifs of the frequently sequenced domains IV and V of the insect mitochondrial large subunit rRNA gene
---
authors: 'Bulyk, Martha L'
title: Computational prediction of transcription-factor binding site locations
---
authors: 'Bulyk, Martha L. and Johnson, Philip L. F. and Church, George M.'
title: Nucleotides of transcription factor binding sites exert interdependent effects on the binding affinities of transcription factors
---
authors: 'Bundschuh, R. and Hwa, T.'
title: An analytic study of the phase transition line in local sequence alignment with gaps
---
authors: 'Buratti, Emanuele and Muro, Andres F. and Giombi, Maurizio and Gherbassi, Daniel and Iaconcig, Alessandra and Baralle, Francisco E.'
doi: 10.1128/MCB.24.3.1387-1400.2004
title: RNA Folding Affects the Recruitment of SR Proteins by Mouse and Human Polypurinic Enhancer Elements in the Fibronectin EDA Exon
---
authors: 'Burgar, Helen R. and Burns, Helen D. and Elsden, Joanna L. and Lalioti, Maria D. and Heath, John K.'
doi: 10.1074/jbc.M107785200
title: Association of the Signaling Adaptor FRS2 with Fibroblast Growth Factor Receptor 1 (Fgfr1) Is Mediated by Alternative Splicing of the Juxtamembrane Domain
---
authors: 'Burge, Chris and Karlin, Samuel'
title: Prediction of Complete Gene Structures in Human Genomic DNA
---
authors: 'Burge, Christopher B. and Padgett, Richard A. and Sharp, Phillip A.'
title: Evolutionary Fates and Origins of U12-Type Introns
---
authors: 'Burgler, Craig and Macdonald, Paul M'
doi: 10.1186/1471-2164-6-88
title: 'Prediction and verification of microRNA targets by MovingTargets, a highly adaptable prediction method'
---
authors: 'Burke, John and Wang, Hui and Hide, Winston and Davison, Daniel B.'
title: Alternative Gene Form Discovery and Candidate Gene Selection from Gene Indexing Projects
---
authors: 'Burley, Stephen K and Kamada, Katsuhiko'
title: Transcription factor complexes
---
authors: 'Burset, M. and Seledtsov, I. A. and Solovyev, V. V.'
title: Analysis of canonical and non-canonical splice sites in mammalian genomes
